gateway/main.go

```diff
@@ -82,6 +82,11 @@ func Run() {
 			log.Fatal(err)
 		}
 
+		_, err = models.CreateDefaultRunbookConfiguration(models.DB, org.ID)
+		if err != nil {
+			log.Fatal(err)
+		}
+
```

Comment:
Also, if it fails creating, should I not run?  @EmanuelJr @sandromello

Reply:
I think it should be a best effort scenario